README.md

```diff
@@ -11,6 +11,42 @@
 
 This package allows us to reuse GitHub actions, which simplifies version management.
 
+### Example of use of the action [Composer require checker](https://github.com/maglnet/ComposerRequireChecker)
+
+```yml
```

Comment:
Maybe move this to separate folder and file and refer here with a link? What do you think?

Reply:
Yes, but let's do it in a separate PR.